extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -20,11 +20,11 @@
     public Optional<String> authServerUrl = Optional.empty();
 
     /**
-     * Enables OIDC discovery.
+     * Enables OIDC discovery. Default value is 'true'.
```

Comment:
You have the very useful `defaultValueDocumentation` for that which is printed nicely as the default value in the config reference doc.